task.go

```diff
@@ -236,7 +248,12 @@ func (e *Executor) runCommand(ctx context.Context, t *taskfile.Task, call taskfi
 
 	switch {
 	case cmd.Task != "":
-		return e.RunTask(ctx, taskfile.Call{Task: cmd.Task, Vars: cmd.Vars})
+		err := e.RunTask(ctx, taskfile.Call{Task: cmd.Task, Vars: cmd.Vars})
+		if err != nil {
+			e.Logger.Errf("%s", err)
```

Comment:
Same here.

This log isn't needed. If an error happen, it'll be returned and logged by the caller.